include/image_undistort/depth.h

```diff
@@ -26,17 +26,26 @@ namespace image_undistort {
 constexpr int kQueueSize = 10;
 // small number used to check things are approximately equal
 constexpr double kDelta = 0.000000001;
-// stereobm parameters
-const std::string kPreFilterType = "xsobel";
-constexpr int kPreFilterSize = 9;
+// stereo parameters
 constexpr int kPreFilterCap = 31;
 constexpr int kSADWindowSize = 11;
 constexpr int kMinDisparity = 0;
 constexpr int kNumDisparities = 64;
-constexpr int kTextureThreshold = 0;
 constexpr int kUniquenessRatio = 0;
 constexpr int kSpeckleRange = 3;
 constexpr int kSpeckleWindowSize = 500;
+// bm parameters
+constexpr int kTextureThreshold = 0;
+const std::string kPreFilterType = "xsobel";
+constexpr int kPreFilterSize = 9;
+// sgbm parameters
+constexpr bool kUseSGBM = false;
+constexpr int kP1 = 120;
+constexpr int kP2 = 240;
+constexpr int kDisp12MaxDiff = -1;
+constexpr bool kUseHHMode = false;
```

Comment:
Do you know if these defaults make sense for the vi-sensor? Probably make sense that our defaults make sense for that. I'm looking into this now.